resources/prometheus/prometheus-rules.yaml

```diff
@@ -291,3 +291,139 @@ spec:
               The DB instance `{{ $labels.dimension_DBInstanceIdentifier }}` has scaled up as high as it can.
               Consider increasing the maximum ACU setting for the cluster.
             sop_url: "https://github.com/stackrox/acs-managed-service-runbooks/blob/master/sops/dp-014-aws-rds-acu-utilization.md"
+
+    - name: rhacs-central.sli
```

Comment:
It would be nice have the `rhacs-*` labels on the recording rules. In particular, `rhacs_instance_id`, `rhacs_org_name`, `rhacs_org_id`, `rhacs_cluster_name`. This probably requires some join

Reply:
What about `rhacs_environment`? Does it make sense to include it here?

Reply:
Yes that one as well.